src/utils_rmw.c

```diff
@@ -143,3 +143,28 @@ human_readable_size (off_t size)
 
   return buffer;
 }
+
+/*!
+ * Verify with the user about an action. The prompt produced will be
+ * "Do you want to continue? (y/n): ". If the user provides "y",
+ * then 1 is returned, else 0 is returned.
+ * @return bool
+ */
+bool
+user_verify (void)
+{
+  printf("Do you want to continue? (y/n): ");
+  char answer;
```

Comment:
This function could be simplified:

```
	char answer = getchar ();
	return (answer == 'y' || answer == 'Y');
```

Reply:
If you write it so that `getchar()` is called only once, any characters after the first char is left behind in the stdout buffer (including the newline when the user verifies their response). This can cause errors later if the buffer is used again. The way I've written it means the buffer is flushed and the user MUST type exactly 'y' (e.g. 'yfhfpwfhew' is not valid even though the first char is 'y').

Side note, I notice your version accepts 'Y' or 'y'. Would you prefer this over just accepting 'y' (which is what my code does)?

Reply:
ok, that sounds better. I'll play with the code a little so I understand it better.

It may need a little tweaking.. see https://app.codacy.com/app/andy5995/rmw/pullRequest?prid=3788060 not that we have to trust what codacy says but I'll play with the code a little.

One change though is that the variable being returned can't be named "yes". (code clarity)

Reply:
> Side note, I notice your version accepts 'Y' or 'y'. Would you prefer this over just accepting 'y' (which is what my code does)?

Yes please :)